### ISSUE TITLE
Release/Tag new version (and use semantic versioning)

### ISSUE THREAD

**Issue by rjhilgefort** [jeromeetienne/jquery-qrcode#76]
I'm currently maintaining a fork just so I can more frequently version this codebase. There hasn't been a tag/release in about 6 months. Tagging every so often (after big pull requests, features, etc...) would be a big help for using this as a dependency. So please make `v1.1` or...

I also tend to prefer [semantic versioning (semver)](http://semver.org/) for versioning projects. This isn't a huge deal, but just thought I'd mention it. If you were to fall in line with semver, your new version would probably be `v1.0.1` unless you feel that the updates that have been made since `v1.0` are significant enough to warrant bumping to `v1.1.0`. It's your call (obviously).

**TL;DR** Push out a new release/version please. Thanks for the awesome lib!

**Comment by AO16** [comment]
:+1:

**Comment by Afterster** [comment]
+1, @jeromeetienne?
